aidial_adapter_bedrock/llm/model/claude/v3/converters.py

```diff
@@ -219,7 +218,9 @@ def to_dial_finish_reason(
                 case ToolsMode.FUNCTION_EMULATION:
                     return FinishReason.FUNCTION_CALL
                 case _:
-                    assert_never(tools_mode)
+                    raise Exception(
+                        f"Invalid tools mode {tools_mode} during tool use!"
```

Comment:
This will always print `Invalid tools mode None during tool use!`.

Maybe something more telling: 

```python
  case None:
    raise RuntimeException("A model has called a tool, but no tools were given to the model in the first place.")
```

Reply:
Fixed

Reply:
case AIFunctionCallMessage() | HumanFunctionResultMessage():
                raise ValueError(
                    "Function messages should be converted to tool messages before sending to Claude!"
                )
 ```
 
 
 It will be similar to error above for SystemMessages
 ```python
            case SystemMessage():
                raise ValueError(
                    "System message is only allowed as the first message"
                )
```